package ar.edu.utn.frba.dds.services.exceptions;

public class PersonaHumanaNoEncontradaException extends RuntimeException  {
}
